android/app/src/main/java/com/votlin/android/ui/fragment/TalksListFragment.kt

```diff
@@ -41,56 +41,52 @@ class TalksListFragment : RootFragment<TalksListView>(), TalksListView {
     override val fragmentModule: Kodein.Module = Kodein.Module {
         bind<TalksListPresenter>() with provider {
             TalksListPresenter(
-                    view = this@TalksListFragment,
-                    errorHandler = AndroidErrorHandler(),
-                    executor = instance(),
-                    repository = instance())
+                view = this@TalksListFragment,
+                errorHandler = AndroidErrorHandler(),
+                executor = instance(),
+                repository = instance()
+            )
         }
     }
 
-    private val adapter = TalksAdapter { presenter.onTalkClicked(it) }
+    private lateinit var talksAdapter: TalksAdapter
```

Comment:
Why this var should be lateinit? We must avoid lateinit every time we could. Also, in that case, talks adapter must be readonly because it must be always the same adapter

Reply:
yeah, right, that's wrong. I was thinking about instantiating the adapter as local val in the method for initializing UI, but not as instance variable. My bad, going to change it.